src/foam/u2/tag/CircleIndicator.js

```diff
@@ -129,6 +133,15 @@ foam.CLASS({
         .end();
       }
 
+      if ( this.indicateProcessing ) {
+        this.start(this.LoadingSpinner)
+          .style({
+            'width': '' + this.size + 'px',
```

Comment:
Is the '' + needed?

Reply:
Thanks - it appears there was a little bit of witch-doctoring in there